src/ai/defaultai.h

```diff
@@ -383,9 +383,9 @@ struct DefaultAI : ComputerPlayer {
 
 	int16_t productionsites_ratio_;
 
-	bool resource_necessity_water_needed_{false};  // unless atlanteans or amazons
+	bool resource_necessity_water_needed_{false};  // unless the Atlanteans or the Amazons
```

Comment:
I don't think this needs the "the"-s. OTOH it should probably be rephrased. @hessenfarmer?

Reply:
```suggestion
	bool resource_necessity_water_needed_{false};  // unless we are Atlanteans or Amazons
```